app/src/main/java/net/skyscanner/backpack/demo/compose/ThemeStory.kt

```diff
@@ -105,7 +106,7 @@ fun ThemeStory() {
       )
     }
 
-    CircularProgressIndicator()
+    BpkSpinner(size = BpkSpinnerSize.Medium)
```

Comment:
We've been removing the examples from here, rather than replacing them with the bpk version as this is showcasing the theming of material components